.github/actions/compute-sdk-test/main.js

```diff
@@ -80,19 +117,25 @@ const mainAsyncTask = async () => {
 
   // Iterate through the module tests, and run the Viceroy tests
   for (const moduleKey of moduleKeys) {
+    const testBase = `${fixtureBase}/${moduleKey}`;
+
+    // created/used by ./integration-tests/js-compute/build-one.sh
+    const fastlyTomlPath = `${testBase}/fastly.toml`;
+    const wasmPath = `${testBase}/${moduleKey}.wasm`;
+    const pkgPath = `${testBase}/pkg/${moduleKey}.tar.gz`;
+
     const module = modules[moduleKey];
     const moduleTestKeys = Object.keys(module.tests);
     console.info(`Running tests for the module: ${moduleKey} ...`);
 
     // Spawn a new viceroy instance for the module
     viceroy = new Viceroy();
     const viceroyAddr = '127.0.0.1:8080';
-    await viceroy.spawn(module.wasm_path, {
-      config: module.fastly_toml_path,
+    await viceroy.spawn(wasmPath, {
+      config: fastlyTomlPath,
```

Comment:
Can't this use the same `spawnViceroy` function as the backend does? That function already needs to match up with `build-one.sh` so I think you should be able to get rid of the `fastlyTomlPath` and `wasmPath` declarations here.